docs/1.getting-started/6.data-fetching.md

```diff
@@ -16,9 +16,9 @@ Both `useFetch` and `useAsyncData` share a common set of options and patterns th
 
 Before that, it's imperative to know why these composables exist in the first place.
 
-## Why using specific composables?
+## Why use specific composables for data fetching?
 
-When using a framework like Nuxt that can perform calls and render pages on both client and server environments, some challenges must be addressed. This is why Nuxt provides composables to wrap your queries, instead of letting the developer rely on [`$fetch`](/docs/api/utils/dollarfetch) calls alone.
+Nuxt is a framework which can run isomorphic, or universal code, in both server and client environments. If a normal `fetch` call is used with javascript, data will be fetched twice, once on the server and once on the client. This is why Nuxt offers specific data fetching composables so data is fetched once on the server, and not a second time on the client.
```

Comment:
```suggestion
Nuxt is a framework which can run isomorphic (or universal) code in both server and client environments. If the [`$fetch` function](/docs/api/utils/dollarfetch) is used to perform data fetching in the setup function of a Vue component, this may cause data to be fetched twice, once on the server (to render the HTML) and once again on the client (when the HTML is hydrated). This is why Nuxt offers specific data fetching composables so data is fetched only once.
```